src/erldns_resolver.erl

```diff
@@ -20,6 +20,7 @@
 -include("erldns.hrl").
 
 -export([resolve/3]).
+-export([best_match/2]).
```

Comment:
I wonder if `best_match/2` should be moved to `erldns_zone_cache` in a future PR?

Reply:
yes, probably better to move them closer to the source of data used for resolution decisions: https://github.com/dnsimple/erldns/issues/138